test/acceptance/helpers/kubectl.go

```diff
@@ -29,7 +29,7 @@ func RunKubectlAndGetOutputWithLoggerE(t testing.TestingT, options *k8s.KubectlO
 	if options.ConfigPath != "" {
 		cmdArgs = append(cmdArgs, "--kubeconfig", options.ConfigPath)
 	}
-	if options.Namespace != "" {
+	if options.Namespace != "" && !sliceContains(args, "-n") && !sliceContains(args, "--namespace") {
```

Comment:
@ishustava this is a bit hacky but might save someone who is setting `-n` and not creating their own kube ops. 👍 / 👎 ?

Reply:
Oh yeah, this was a bug that I fixed, but haven't merged. I'm populating the namespace correctly in the Kubectl options:
https://github.com/hashicorp/consul-helm/blob/88452928f7433170f23952a03b62668b748046c5/test/acceptance/framework/environment.go#L79-L102

Let me know if you want me to merge it before this PR. Sorry about that, that must have been confusing to figure out!

Reply:
This will work if you set the namespace in your kubecontext, otherwise, you will need to run tests with the `-namespace` flag.

Reply:
I left some comments and deleted them because I need to learn how to read the code first! Yeah, that looks good to me.

Reply:
lol, I thought github was broken for a sec